shared/src/main/scala/amf/client/remod/AMFResolver.scala

```diff
@@ -5,8 +5,8 @@ import amf.core.model.document.BaseUnit
 
 private[remod] object AMFResolver {
```

Comment:
The name could be AMFTransformer to conform with the internal name change. Also it avoids confusion with AMFResolvers that is the class that contains the ResourceLoader and other things.